client/ayon_equalizer/hooks/pre_add_last_workfile_arg.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+from ayon_applications import LaunchTypes, PreLaunchHook
+
+
+class AddLast3DEWorkfileToLaunchArgs(PreLaunchHook):
+    """Add last workfile path to launch arguments.
+
+    Copied from ayon_core/hooks/pre_add_last_workfile_arg.py.
+    Checks 'start_last_workfile', if set to False, it will not open last
+    workfile. This property is set explicitly in Launcher.
+    """
+
+    # Execute after workfile template copy
+    order = 10
+    app_groups = ("equalizer", "sdv_3dequalizer")
+    launch_types = LaunchTypes.local
```

Comment:
This must be a `set`
```suggestion
    launch_types = {LaunchTypes.local}
```

Reply:
I copied it from https://github.com/ynput/ayon-equalizer/blob/abf6ea374bb0034e70d5ceb9570bffc002f22353/client/ayon_equalizer/hooks/pre_install_qt_bindings.py#L28 so this needs to be changed too?

Reply:
I'd say so. @antirotor any idea where that `sdv_3dequalizer` comes from?

Reply:
I think it comes from the time where you couldn't have number at the start of the name, so it was prefixed by "sdv" -Science D-Visions". Then it stayed there for compatibility reasons